docs/source/getting_started/project_structure.rst

```diff
@@ -16,47 +19,51 @@ The repository directory (or root directory) contains the following:
 
 - ``docs/``
 
-  - This directory contains the documentation for the repository (which includes
-    the file you are reading now).
+  - This directory contains the documentation for the repository (which includes the file you are reading now).
 
 - ``subtitles/``
 
   - This directory contains subtitle files for CS Field Guide videos.
 
 - ``README.md``
 
-  - This file contains an introduction and important information for the
-    repository.
+  - This file contains an introduction and important information for the repository.
 
 - ``LICENCE.md``
 
   - This file details the licences the repository uses.
```

Comment:
How about:
This file details the ``MIT licence`` that covers the project infrastructure. Project content is covered by ``LICENCE-CONTENT``, and third party libraries used are listed in ``LICENCE-THIRD-PARTY``, with a full copy of their respective licences in the ``third-party-licences`` directory.

Also none of these files are markdown, so remove the `.md`